Formula/camlp4.rb

```diff
@@ -35,5 +36,8 @@ def install
                             "foo.ml", "-o", testpath/"foo.ml.out"
     assert_equal "type t = [ Homebrew | Rocks ];",
                  (testpath/"foo.ml.out").read.strip
+
+    (testpath/"try_camlp4.ml").write "open Camlp4"
+    system "ocamlc", "-c", "-I", "/usr/local/lib/ocaml/camlp4", "-o", testpath/"try_camlp4.cmo", testpath/"try_camlp4.ml"
```

Comment:
Since `/usr/local` isn't the path for everyone, could you write it like in the install block.

```suggestion
    system "ocamlc", "-c", "-I", "#{HOMEBREW_PREFIX}/lib/ocaml/camlp4", "-o", testpath/"try_camlp4.cmo", testpath/"try_camlp4.ml"
```